app/screens/PresentationPreviewScreen/PresentationPreviewScreen.tsx

```diff
@@ -30,6 +40,88 @@ export default function PresentationPreviewScreen({
     );
   }
 
+  async function handleButtonPress() {
+    if (isCreateLinkMode && selectedCredentials.length > 0) {
+      const rawCredentialRecord = selectedCredentials[0];
+      
+      // Check if credential is verified
+      const verifyPayload = await verificationResultFor({rawCredentialRecord, registries});
+      const verified = verifyPayload?.verified;
+      
+      if (!verified) {
+        await displayGlobalModal({
+          title: 'Unable to Create Link',
```

Comment:
Didn't we disable this logic for the other pathway? (Meaning, I'm fairly sure you can create public links now for not-verified credentials)